microcloud/test/includes/microcloud.sh

```diff
@@ -520,7 +538,8 @@ reset_system() {
 
     reset_snaps "${name}"
 
-    lxc exec "${name}" -- zpool destroy -f local || true
+    # lxc exec may lock up on fast-running commands if running in the background, so we need to sleep 1s.
+    lxc exec "${name}" -- sh -c "sleep 1 ; zpool destroy -f local" || true
```

Comment:
Moving `sleep 1` out of the `lxc exec` would avoid needing this `sh -c`.

Reply:
If I remove it out of the `lxc exec` then it would defeat the purpose. It's the `lxc exec` itself returning too quickly which causes the issue.

Reply:
OK, that's embarrassing that I missed the whole point :/
Thanks :)